### DISCUSSION TITLE
Неадекватное поведение в WEB режиме

### DISCUSSION THREAD

**Discussion by RRozi** [flet-dev/flet#2963] [Q&A]
При работе в WEB, с использованием виджетов DatePicker и TimePicker появляются проблемы(Без них всё супер). Мною конкретно было замечено 
- Проблемы со SnackBar
- Текст может просто тупо появится сверху в куче, по вверх всех виджетов
- Проблемы с обновлением страницы
То есть если DatePicker и TimePicker используются в проекте, то начинают проявляться подобные проблемы, как минимум те что я заметил.

**Comment by FeodorFitsner** [comment]
Post issues in English please.
